### PR TITLE
Added module on damage analysis for Eearthquakes

### PR DESCRIPTION
**Pull request by Claurt07**
Added module developed by Honey on damage analysis for Earthquakes as module 7.4 (only in English)

### CODE REVIEW

File: content/en/pages/data-use-and-analysis/7.5-OSM-Analytics
Comment (dakotabenjamin) [inline content/en/pages/data-use-and-analysis/7.5-OSM-Analytics:1]: It looks like this file was never registered in the menu- could you add `.md` to the end of it? and rename the header as well

File: static/images/data-use-and-analysis/fig1_damage_analysis.jpeg
Comment (dakotabenjamin) [inline static/images/data-use-and-analysis/fig1_damage_analysis.jpeg:1]: Not a blocker for this PR to be merged- but these days for web-served content, .webp or .png are preferred formats.

**Review by dakotabenjamin — APPROVED** [review]

File: static/images/data-use-and-analysis/fig1_damage_analysis.jpeg
Comment (emi420) [reply static/images/data-use-and-analysis/fig1_damage_analysis.jpeg:1]: @Claurt07 Screenshots will look much better if you use PNG :)

JPEG are better for digital photos and PNG for web graphics.

**Review by emi420 — CHANGES_REQUESTED** [review]
Hey @Claurt07 could you change images from JPEG for PNG, add .md to `content/en/pages/data-use-and-analysis/7.5-OSM-Analytics` and rename the header? thanks!

File: static/images/data-use-and-analysis/fig1_damage_analysis.jpeg
Comment (Claurt07) [reply static/images/data-use-and-analysis/fig1_damage_analysis.jpeg:1]: I was trying to use PNG, only JPEG would load/render. As I had to release this ASAP I did not explore why PNG was not loading (if you check all the other files are either JPEG or GIF). 
For future ones I will explore how to use PNG instead

File: content/en/pages/data-use-and-analysis/7.5-OSM-Analytics
Comment (Claurt07) [reply content/en/pages/data-use-and-analysis/7.5-OSM-Analytics:1]: This document was already there, so I did not delete, only changed the number from 7.4 to 7.5 so the new module would take its place. As it seems its incomplete I did not change it to register on the menu.

**Comment by Claurt07** [comment]
Hey, @emi420 just echoing the above comments:
- I tried using PNG, but only JPEG would load.  All the other files on the ToolBox are either JPEG or GIF.
- The page "OSM Analytics" was already there, so I did not delete it, I only changed the number from 7.4 to 7.5 so the new module would take its place. As it seems incomplete, I did not change it to register on the menu.

**Pull request merged**